### PR TITLE
Upgrade frontend to Ember 3.0.0

### PR DESCRIPTION
**Pull request by CvX**
Upgrades Ember to version 3.0.0.

### CODE REVIEW

**Comment by Turbo87** [comment]
As mentioned on the Ember.js Slack I would prefer it if the PR could be broken apart into multiple PRs to make review a little easier. My suggestion would be to start with the new testing API changes, then the native-dom-helpers and finally the upgrade itself.

File: package.json [hunk 1]
```diff
@@ -46,6 +46,7 @@
     "ember-cli-shims": "^1.2.0",
     "ember-cli-sri": "2.1.1",
     "ember-cli-uglify": "^2.0.0",
+    "ember-cli-update": "^0.15.0",
```
Comment (Turbo87) [inline package.json]: I'd prefer to use the global installation of this, since we don't need to install it for CI systems etc.

Comment (CvX) [reply package.json]: Alright, commit removed.

**Comment by Turbo87** [comment]
@CvX could you extract the ESLint changes to a separate PR too? Ideally all that remains in this PR will be the `ember-source` version bump :)

**Pull request closed without merging**